src/applications/healthcare/questionnaire/sass/review.scss

```diff
@@ -0,0 +1,16 @@
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
+@import "~@department-of-veterans-affairs/formation/sass/modules/m-process-list";
+@import "~@department-of-veterans-affairs/formation/sass/modules/m-form-process";
+@import "~@department-of-veterans-affairs/formation/sass/modules/m-progress-bar";
+@import "../../../../platform/forms/sass/m-schemaform";
+@import "~@department-of-veterans-affairs/formation/sass/modules/m-modal";
+@import "~@department-of-veterans-affairs/formation/sass/modules/m-omb-info";
+@import "../../../../platform/forms/sass/m-form-confirmation";
+
+
+
+.healthcare-questionnaire-review{
+  div.top-row.review-row{
+    border-top: none;
+  }
+}
```

Comment:
Do we care about adding a new line to the end of certain files?

Reply:
Our linter isn't set up for SCSS files yet, so there is no standard. I'll make it consistent for our files.